cpu/ppc64_cpu_test.py

```diff
@@ -147,6 +152,14 @@ def dscr(self):
         command2 = "cat /sys/devices/system/cpu/dscr_default"
         self.equality_check("DSCR", command1, command2)
 
+    def smt_loop(self):
+        """
+        Tests smt on/off in a loop
+        """
+        for i in range(1, 100):
+            process.system_output("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
```

Comment:
smt off operation takes longer time. The time depends on number of cpus. Running them together will ensure that smt on will not be executed before smt off is completed. This will avoid any timing errors